benchmark/runner/mini_runners.cpp

```diff
@@ -1068,7 +1324,7 @@ BENCHMARK_DEFINE_F(MiniRunners, SEQ2_0_IndexScanRunners)(benchmark::State &state
   std::stringstream query;
   auto table_name = execution::sql::TableGenerator::GenerateTableIndexName(type, num_rows);
```

Comment:
I forgot that why for some runners we're using this set of tables and for other runners we're using the default set of tables? The comments for `GenerateMiniRunnersData` and `GenerateMiniRunnerIndexTables` in `TableGenerator` are very light in terms of explaining their differences.

Reply:
I think this is a legacy decision (I don't remember why this was the case either). Either way, I have removed `GenerateMiniRunnerIndexTables` and unified it such that all runners use the same tables.